lib/main_tab_widget.dart

```diff
@@ -1,5 +1,10 @@
 import 'package:flutter/cupertino.dart';
 import 'package:flutter/material.dart';
+import 'package:scbsss/models/journal_entry.dart';
```

Comment:
Not quite sure why but based on the file organization we have in our repo, this doesn't work... I changed it in my branch but from what I know it should be: "import 'models/main_tab_widget.dart';". At least that's what worked on my machine... Not sure why I need to create a different path. But regardless, we can make a fix for this

Reply:
Never mind! Just figured out the issue. The file works just fine with these imports.